head/src/main/kotlin/io/qalipsis/core/head/campaign/PersistentCampaignService.kt

```diff
@@ -42,6 +42,7 @@ import io.qalipsis.core.head.utils.SortingUtil
 import io.qalipsis.core.head.utils.SqlFilterUtils.formatsFilters
 import jakarta.inject.Singleton
 import java.time.Instant
+import kotlin.math.E
```

Comment:
???

Reply:
It was a mistake, I don't know what happened

Reply:
It was a mistake, I don't know how it happened.